MIGRATION.md

```diff
@@ -18,7 +18,20 @@
 
 ## From version 3.2.x to 3.3.x
 
-There should be no breaking changes in this release, but read on if you're using `addon-knobs`: we advise an update to your code for efficiency's sake.
+There wasn't expected be any breaking changes in this release, but unfortunately it turned out that there are some. We're revisiting our [release strategy](https://github.com/storybooks/storybook/blob/master/RELEASES.md) to follow semver more strictly.
```

Comment:
TODO: replace "we're revisiting" with "we've revisited" once we update the `RELEASES.md` file